desktop/src/main/java/bisq/desktop/main/offer/MutableOfferViewModel.java

```diff
@@ -729,7 +729,7 @@ void onFocusOutAmountTextField(boolean oldValue, boolean newValue) {
             } else if (amount.get() != null && btcValidator.getMaxTradeLimit() != null && btcValidator.getMaxTradeLimit().value == OfferRestrictions.TOLERATED_SMALL_TRADE_AMOUNT.value) {
                 amount.set(btcFormatter.formatCoin(btcValidator.getMaxTradeLimit()));
                 new Popup().information(Res.get("popup.warning.tradeLimitDueAccountAgeRestriction.buyer",
-                        btcFormatter.formatCoinWithCode(OfferRestrictions.TOLERATED_SMALL_TRADE_AMOUNT),
+                        btcFormatter.formatCoinWithCode(getEffectiveLimit()),
```

Comment:
this looks fine I guess. Was it wrong in bisq?
Note that this is displayed only when `getMaxTradeLimit == TOLERATED_SMALL_TRADE_AMOUNT` but I don't quite understand what is MaxTradeLimit and what is MaxValue

The message here is: 
> The allowed trade amount is limited to {0} because of security restrictions based on the following criteria:\n\ - The buyer''s account has not been signed by an arbitrator or a peer\n\ - The time since signing of the buyer''s account is not at least 30 days\n\ - The payment method for this offer is considered risky for bank chargebacks

The message looks like it's about the limit that is reduced if the account isn't signed

Reply:
> Was it wrong in bisq?

Bisq displays the max configured trade limit in the popup instead of the effective limit with deductions, so the user input can be rejected even if they heed the warning to reduce the amount.

Reply:
> but I don't quite understand what is MaxTradeLimit and what is MaxValue

MaxTradeLimit is the maximum configured limit.  MaxValue is the maximum effective limit if there were any reductions, null otherwise.  It's confusing, hence separating this check to its own method.